Regression of pylint config - import-error's are no longer disabled

## Describe the bug

PR #3038 removed the option `import-error` from the `TEMPLATES/.python-lint` file and has caused pylint to throw up import errors. This removal seems accidental as the PR was to remove deprecated/removed options (Address issue #3037).
The `import-error` option was added back in PR #695 to deal with #157 and #645

## Expected behaviour

Import errors aren't reported by pylint inside of the super-linter. i.e. the following error shouldn't show up:
`E0401: Unable to import '*******' (import-error)`

## Steps to Reproduce

1. Create a python file with 3rd-party library imports (e.g. pytest)
```shell
echo 'import pytest' > /tmp/test.py
```

2. Run the super linter
```
docker run \
-e RUN_LOCAL=true \
-e USE_FIND_ALGORITHM=true  \
-e VALIDATE_PYTHON_PYLINT=true \
-v /tmp/test.py:/tmp/lint/test.py \
github/super-linter:v4
```

3. See logs

## Logs

```text
2022-07-28 01:50:53 [INFO]   Linting [PYTHON_PYLINT] files...
2022-07-28 01:50:53 [INFO]   ----------------------------------------------
2022-07-28 01:50:53 [INFO]   ----------------------------------------------
2022-07-28 01:50:53 [INFO]   ---------------------------
2022-07-28 01:50:53 [INFO]   File:[/tmp/lint/test.py]
2022-07-28 01:50:54 [ERROR]   Found errors in [pylint] linter!
2022-07-28 01:50:54 [ERROR]   Error code: 2. Command output:
------
************* Module test
test.py:1:0: E0401: Unable to import 'pytest' (import-error)
------
2022-07-28 01:50:55 [INFO]   ----------------------------------------------
2022-07-28 01:50:55 [INFO]   ----------------------------------------------
```

## Additional context

--- a/TEMPLATES/.python-lint
+++ b/TEMPLATES/.python-lint
@@ -62,6 +62,7 @@ disable=raw-checker-failed,
         suppressed-message,
         useless-suppression,
         deprecated-pragma,
+        import-error
 
 # Enable the message, report, category or checker with the given id(s). You can
 # either give multiple identifier separated by comma (,) or put this option

